packages/website/docusaurus.config.mts

```diff
@@ -91,6 +91,12 @@ const themeConfig: AlgoliaThemeConfig & ThemeCommonConfig = {
             label: 'Twitter',
             rel: 'me noopener',
           },
+          {
+            className: 'image-link bluesky-link social-link-icon',
+            href: 'https://bsky.app/profile/typescript-eslint.io',
+            label: 'Bluesky',
+            rel: 'me noopener',
+          },
```

Comment:
These were sorted alphabetically, but now they're not - could you put Bluesky at the top of the list please?

Reply:
Thank you for the review! 😊